### PR TITLE
Hardening/refactor repo only cygnus

### PR DESCRIPTION
**Pull request by fgalan**
This PR should be merged after merging https://github.com/telefonicaid/fiware-livedemoapp/pull/21

@frbattid

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,565 @@
-# FI-WARE Connectors
+# Cygnus connector
```
Comment (fgalan) [inline README.md]: Regarding root README, what I have done is to delete the old one and move the older flume/README.md to the root directory. I think the only think we "lost" is the reference to resources/ directory, but I think it doesn't mind.

File: doc/devel/add_new_sink.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ We will not merge new code in the Cygnus repository coming from a different path
 
 ### Coding style 
 
-Please, add the `fiware-connectors/flume/telefonica_checkstyle.xml` to you IDE as check style configuration. This XML file contains all the coding style rules accepted by Telefónica.
+Please, add the `fiware-cygnus/telefonica_checkstyle.xml` to you IDE as check style configuration. This XML file contains all the coding style rules accepted by Telefónica.
```
Comment (fgalan) [inline doc/devel/add_new_sink.md]: I have remove some 'flume/' references in the doc/, but probably there are some that I haven't spoted or are situated in other directories.

Similarly, I have changed the repository name in docs for fiware-connectors to fiware-cygnus, anticipating the new name at the end.

File: README.md (hunk 1 above)
Comment (frbattid) [inline README.md]: NTC

File: doc/devel/add_new_sink.md (hunk 2 above)
Comment (frbattid) [inline doc/devel/add_new_sink.md]: I'll perform an exhaustive search when implementing the issue https://github.com/telefonicaid/fiware-connectors/issues/294

NTC

**Comment by frbattid** [comment]
LGTM

**Pull request merged**